src/libos/src/entry.rs

```diff
@@ -87,6 +93,39 @@ pub extern "C" fn occlum_ecall_init(log_level: *const c_char, instance_dir: *con
         unsafe { backtrace::enable_backtrace(&ENCLAVE_PATH, PrintFormat::Short) };
     });
 
+    // Read resolv.conf file from host
+    let resolv_conf_bytes = unsafe {
+        assert!(!resolv_conf_ptr.is_null());
+        CStr::from_ptr(resolv_conf_ptr).to_bytes()
+    };
+
+    // Parse and inspect resolv.conf file
+    match resolv_conf::Config::parse(resolv_conf_bytes) {
```

Comment:
Here is a logical problem, if the copied resolv.conf is malformatted, we think it it appropriate to make occlum crash, so we decide to drop the malformatted resolv.conf rather than return when parsing error appear.

Reply:
Done